R/prep_scores_scorecard.R

```diff
@@ -47,3 +47,106 @@ prep_scores_scorecard <- function(results_portfolio,
   }
   data_out
 }
+
+#' Prepare share of portfolio emissions covered by aggregate score analysis
+#'
+#' @param emissions_data Data frame that contains pre-wrangled emissions data
+#' @param log_dir Path to log file
+#'
+#' @return numeric
+#' @export
+prep_scores_emissions_scorecard <- function(emissions_data,
+                                            log_dir) {
```

Comment:
If you add an arg `log_dir` here, you will need to add that argument to every place it is called (e.g. in the `.Rmd` template). 

I think the safest approach would be to add the arg, with a default value of `GlobalEnv$log_dir`:

``` r
prep_scores_emissions_scorecard <- function(emissions_data, log_dir = GlobalEnv$log_dir) {
```

Reply:
I did add it everywhere it is used, but happy to add the default. would it not be `.GlobalEnv$log_dir` however?

Reply:
done